packages/cli/src/LoadNodesAndCredentials.ts

```diff
@@ -428,19 +429,23 @@ class LoadNodesAndCredentialsClass {
 	}
 
 	/**
-	 * Retrieves `categories`, `subcategories` and alias (if defined)
-	 * from the codex data for the node at the given file path.
+	 * Retrieves `categories`, `subcategories`, partial `resources` and
+	 * alias (if defined) from the codex data for the node at the given file path.
 	 *
 	 * @param {string} filePath The file path to a `*.node.js` file
 	 * @returns {CodexData}
 	 */
 	getCodex(filePath: string): CodexData {
 		// eslint-disable-next-line global-require, import/no-dynamic-require, @typescript-eslint/no-var-requires
-		const { categories, subcategories, alias } = require(`${filePath}on`); // .js to .json
+		const { categories, subcategories, resources: allResources, alias } = require(`${filePath}on`); // .js to .json
+
+		const resources = pick(allResources, ['primaryDocumentation', 'credentialDocumentation']);
+
 		// eslint-disable-next-line @typescript-eslint/no-unsafe-return
 		return {
 			...(categories && { categories }),
 			...(subcategories && { subcategories }),
+			...(resources && { resources }),
 			...(alias && { alias }),
 		};
 	}
```

Comment:
My thinking is that in the long run the only maintainable way to maintain the documentation links is to get them directly from the node descriptor (codex). I now exposed `resources.primaryDocumentation` and `resources.credentialDescription` so that the UI can access and use those, instead of having to generate the URLs dynamically from parts.